workers/unity/Packages/io.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -27,6 +27,8 @@ private static readonly string
         // Unix-like: The exit code is 128 + SIGINT (2).
         private const int UnixSigIntExitCode = 128 + 2;
 
+        private static GdkToolsConfiguration toolsConfig;
```

Comment:
Any reason for caching this here when we call `GetOrCreateInstance()` regardless?

Reply:
Removed